### ISSUE TITLE
`is_wrapper(::RFE)` should return `true` not `false`

### ISSUE THREAD

**Issue by ablaom** [JuliaAI/FeatureSelection.jl#15]
```julia
julia> is_wrapper(FeatureSelection.RFE)
false
```

And this line should be removed:

https://github.com/JuliaAI/FeatureSelection.jl/blob/ec0eb11b8b3e34ab12f3486a8a3e8a5b71967aa5/src/models/rfe.jl#L352
```

**Comment by OkonSamuel** [comment]
closed by #18

**Comment by ablaom** [comment]
```julia
julia> using FeatureSelection

julia> using MLJBase

julia> is_wrapper(RecursiveFeatureElimination)
false

(jl_Za1kOv) pkg> st
Status `/private/var/folders/4n/gvbmlhdc8xj973001s6vdyw00000gq/T/jl_Za1kOv/Project.toml`
  [33837fe5] FeatureSelection v0.2.0
  [a7f614a8] MLJBase v1.6.0
```

This is causing the fail [here](https://github.com/JuliaAI/MLJ.jl/pull/1138) and will require another re-release of MLJModels to update the model registry to fix. 

@OkonSamuel
